community/shellinabox/APKBUILD

```diff
@@ -0,0 +1,61 @@
+# Contributor: Ishak BELAHMAR <ishak@nytrio.com>
+# Maintainer: Ishak BELAHMAR <ishak@nytrio.com>
+pkgname=shellinabox
+pkgver=3.9.6
+pkgrel=0
+pkgdesc="publish command line shell through AJAX interface"
+url="https://github.com/shellinabox/shellinabox"
+arch="x86_64"
+license="GPL-2.0"
+depends="zlib musl"
+makedepends="libressl-dev linux-pam-dev zlib-dev autoconf automake libtool g++"
```

Comment:
AL3.9 switched back to openssl, so it is advisable to use that  instead of libressl-dev.